README.md

```diff
@@ -5,4 +5,51 @@
 [![GitHub release](https://img.shields.io/github/release/lacework/terraform-gcp-config.svg)](https://github.com/lacework/terraform-gcp-config/releases/)
 [![CircleCI status](https://circleci.com/gh/lacework/terraform-gcp-config.svg?style=shield)](https://circleci.com/gh/lacework/terraform-gcp-config)
 
-Terraform module for integrating Google Cloud Platform Organziations and Projects with Lacework for cloud resource configuration assessment.
+Terraform module for integrating Google Cloud Platform Organizations and Projects with Lacework for cloud resource configuration assessment.
+
+_**Note:** When `use_existing_service_account` is set to true, the required roles must be added manually. Depending on the integration type this may be at Project or Organization level._
```

Comment:
I am hesitant to change the behavior of all the GCP modules (Config, Audit Log, and GCR),
if we are expecting the users to provide the service account and add the roles manually,
what is the sense of adding this option? If we need to go this route, let us just deprecate it
and have the user use the native Terraform resources for Lacework.

My $0.02 👆🏽